geotrek/signage/views.py

```diff
@@ -95,7 +95,7 @@ def get_queryset(self):
             qs = qs.annotate(api_geom=Transform('geom', settings.API_SRID))
             qs = qs.only('id', 'name', 'published')
         else:
-            qs = qs.select_related('structure', 'manager', 'sealing', 'access', 'type', 'condition')
+            qs = qs.select_related('structure', 'manager', 'sealing', 'access', 'type').prefetch_related('conditions')
```

Comment:
It's good to have `prefetch_related` here, we should do the same thing on the Blade queryset